README.md

```diff
@@ -30,6 +30,30 @@ See [modular architecture section](./docs/guides/modular-architecture.md) for mo
 
 The projects use [npm semantic versioning](https://docs.npmjs.com/about-semantic-versioning)
 
+## Project boards
+
+Between the 5th of June and 17th of July, there is a sprint board used for organising the backlog for each two weeks sprint cycle.
```

Comment:
Super picky date formatting thing - BBC standard is 5 June and 17 July. Just think it's cleaner but it's no biggie :) and maybe change two weeks to two-week.